modules/ROOT/pages/deploy-maven-3.x.adoc

```diff
@@ -81,28 +88,29 @@ This example uses placeholder values. Configure each value with values specific
 
 == Redeploy to Runtime Fabric
 
-To redeploy the application, run the same command as you did to deploy. Runtime Fabric rewrites the application you had deployed.
+To redeploy the application, run the same command as you did to deploy. Runtime Fabric rewrites the 
+application you previously deployed.
 
 == Runtime Fabric Deployment Reference
 
 [%header%autowidth.spread,cols="a,a"]
 |===
 |Parameter | Description
-|runtimeFabricDeployment | specifies a top-level element.
-| uri | specifies your Anypoint Platform URI. +
+|runtimeFabricDeployment | Specifies a top-level element.
+| uri | Specifies your Anypoint Platform URI. +
 If not set, the default value is `+https://anypoint.mulesoft.com+`.
-| muleVersion | specifies the Mule runtime version installed in your Runtime Fabric instance. +
+| muleVersion | Specifies the Mule runtime version installed in your Runtime Fabric instance. +
 Ensure that this value is at equal or higher than the minimum required Mule runtime version of your application.
-| username | specifies your Anypoint Platform username.
-| password | specifies your Anypoint Platform password.
-| applicationName | specifies the name of your application deployed in Exchange. +
+| username | Specifies your Anypoint Platform username.
+| password | Specifies your Anypoint Platform password.
+| applicationName | Specifies the name of your application deployed in Exchange. +
```

Comment:
Exchange is where we store artifacts (like applications). Instead of "deployed", perhaps we update that to "stored".

Then we can reserve the "deploy" term for Runtime Fabric. (In context, when deploying an application to Runtime Fabric using the Maven plugin, the application is stored in Exchange as a prerequisite, implicit step.)

Reply:
Got it - thank you!